viewer/packages/rendering/src/glsl/sector/primitives/cone.frag

```diff
@@ -125,13 +117,17 @@ void main() {
       if (theta < v_angle) theta += 2.0 * PI;
       if (intersectionPoint.z <= 0.0 ||
         intersectionPoint.z > height ||
-        theta > v_angle + v_arcAngle ||
-        isClipped(appearance, p)
+        theta > v_angle + v_arcAngle
       ) {
         // Missed the other point too
         discard;
       }
     }
+    #if NUM_CLIPPING_PLANES > 0
+      if (isClipped(p, v_clippingPlanes)) {
+        discard;
+      }
+    #endif
```

Comment:
Why is this moved?

Reply:
`NUM_CLIPPING_PLANES` value is assigned by the threejs only when clipping is active, else its value is always `0`, so the earlier, the check for discarding the clipped pixel function call will have an compiler error as I am passing `v_clippingPlanes[NUM_CLIPPING_PLANES ]` as a parameter.